### PR TITLE
Feature - Add views and queries for CMO Assets and Operational Requirements to admin

### LINKED ISSUE #99: Pool Dashboard - Create table views for CmoAssets and OperationalRequirements, with API connection
CmoAssets and OperationalRequirements share exactly the same definition: id, key, name and description. Therefore it should be possible to use a common table component, simply passing in different data.

Two (similar) graphql operations must be written, and two wrapper components which load the correct data and pass it to the table component.

### PR DESCRIPTION
**Pull request by mnigh**
Resolves #99.

### CODE REVIEW

File: admin/resources/js/stories/CmoAssetTable.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+});
+const ApiCmoAssetTable = () => {
+  const [result] = useGetCmoAssetsQuery();
+  const { data, fetching, error } = result;
+
+  if (fetching) return <p>Loading...</p>;
+  if (error) return <p>Oh no... {error.message}</p>;
+
+  return <CmoAssetTable cmoAssets={data?.cmoAssets ?? []} />;
+};
```
Comment (tristan-orourke) [inline admin/resources/js/stories/CmoAssetTable.stories.tsx:28]: I would define this in the CmoAssetTable file, and call it CmoAssetTableApi (based on our dev discussion the other day).

File: admin/resources/js/components/CmoAssetTable.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const memoizedData = useMemo(() => cmoAssets.filter(notEmpty), [cmoAssets]);
+
+  return (
+    <>
+      <Table data={memoizedData} columns={columns} />
+    </>
+  );
+};
+
+export default CmoAssetTable;
```
Comment (tristan-orourke) [inline admin/resources/js/components/CmoAssetTable.tsx:39]: Lets also define a component here which wraps CmoAssetTable and includes the urql hook. Both components should be exported, and I'm not sure which should be the default export. Maybe no default export?

**Review by tristan-orourke — CHANGES_REQUESTED** [review]
I would just define the ___Api version of the components in the original table component files.

File: admin/resources/js/components/CmoAssetTable.tsx (hunk 2 above)
Comment (mnigh) [reply admin/resources/js/components/CmoAssetTable.tsx]: I've left out the default export and export each one in CMO Assets and Operational Requirements.

**Review by tristan-orourke — APPROVED** [review]

**Pull request merged**